intro-to-R-tidyverse/01-intro_to_base_R-live.Rmd

```diff
@@ -453,6 +453,47 @@ And view its _levels_ with the `levels()` function.
 levels(iris$Species)
 ```
 
+### Files and directories
```

Comment:
Looking at this in the context of the new exercises pull request...

Should this be H2? It seems to be somewhat separate from the section on data frames.
```suggestion
## Files and directories
```